recipes/RNAtools/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "RNAtools" %}
+{% set version = "0.8.4" %}
+{% set sha256 = "6e6285c54bba3e492581c3b90b2dd86e1f2c2605c6298bee5280fdf0ce19a39b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```

Comment:
Can you add entry points for the following:

https://github.com/grice/RNAtools/blob/master/setup.py#L23-L28

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points

along with tests for the commands:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands

Reply:
Done!